packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -676,6 +677,8 @@ class _RefreshProgressIndicatorPainter extends _CircularProgressIndicatorPainter
     required double strokeWidth,
     required this.arrowheadScale,
   }) : super(
+    // Lengthen the arc a little
+    maxArcLength: 3 / 2 * 1.05, // = 1.575
```

Comment:
That way if the constant changes, all of the places where we use it get updated. :)
```suggestion
    maxArcLength: _kMaxArcLength * 1.05, // = 1.575
```

Reply:
Done